grpc/samples/greeter/Makefile

```diff
@@ -4,11 +4,11 @@ LDFLAGS ?=
 .PHONY: all
 all: server client
 
-greeter_generated.h: greeter.fbs
-	flatc --grpc --cpp $<
+greeter_generated: greeter.fbs
+	flatc --grpc --cpp --filename-suffix "_sample" $<
```

Comment:
I appreciate you want to test this functionality, but let's leave that out, just in case we further complicate matters. For example, the person that reported the breakage was running `flatc` manually, so would have gotten incompatible files also if they didn't know to supply this flag. Let's fix only `idl_gen_grpc.cpp`